### DISCUSSION TITLE
Windows - Rustdesk installed does not popup any login request - If started with "Run only" mode it works

### DISCUSSION THREAD

**Discussion by Flagelmann** [rustdesk/rustdesk#9810] [General]
### Bug Description

On Windows 10/11, when running only Rustdesk client (no installation) I can receive incoming requests from another client and it pops up the "accept incoming" access window.

If I install, on the same Windows 10 system, the Rustdesk software and I try again requesting access from another client it does not pop up any "accept incoming" access window.

I tried with both exe/msi versions and also from an Android device, same behavior.

### How to Reproduce

1. Run-only Rustdesk on Windows 10 in Run-Only mode.
2. Start Rustdesk on another device, configured with the same self-hosted server and the Windows 10 one.
3. Request access to the Windows 10 Run-Only mode endpoint.
4. It pops up the "accept incoming request" window.

1. Redo the same but installing Rustdesk, even the latest version, on a Windows 10/11 system.
2. It should not pops up any "accept incoming request".

### Expected Behavior

It should pops up the "accept incoming request" even when installed and not only when running Rustdesk in "Run only" mode.

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.3.2 -> 1.3.2

### Screenshots

No screens.

**Comment by acrolink** [comment]
I am experiencing the same:
Client: Windows 10 Pro, 22H2. 19045.5131
